src/Evolution/Systems/GeneralizedHarmonic/Characteristics.cpp

```diff
@@ -0,0 +1,191 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+// IWYU pragma: no_include <array>
+
+#include "Evolution/Systems/GeneralizedHarmonic/Characteristics.hpp"
+#include "Characteristics.hpp"
```

Comment:
I'm confused about line 7. Where does this file live? Is this just redundant with line 6?

Reply:
this was a letfover typo from combining `Characteristics.*` and `CharFields.*`. fixed